runtime/compiler/optimizer/EscapeAnalysis.cpp

```diff
@@ -2554,7 +2560,7 @@ bool TR_EscapeAnalysis::collectValueNumbersOfIndirectAccessesToObject(TR::Node *
 
                               if (defNode && defNode->getOpCode().isStore())
                                  {
-                                 if (_vnTemp->get(_valueNumberInfo->getValueNumber(defNode)))
+                                 if (_vnTemp->isSet(_valueNumberInfo->getValueNumber(defNode)))
```

Comment:
What is the thinking behind this change from `get` to `isSet` ?

Reply:
Okay, no need to change. I was just curious since it wasn't a change mentioned in the commit message. But that is fine, thanks.